### DISCUSSION TITLE
AppShell.Header Burger and other controls not working on mobile (iOS/Chrome)

### DISCUSSION THREAD

**Discussion by grahampcharles** [mantinedev/mantine#6643] [Q&A] [answered]
Has anyone had trouble where the burger menu (and other controls in an AppShell.Header) just don't work in production on iOS/mobile? (The links/buttons don't respond at all.)

I'm trying to put together a minimal sample to demonstrate but I thought I'd just toss it out there in case it sounds familiar to anyone.

**Comment by rtivital** [comment, marked as answer]
I'm not aware of any similar issues, provide a sandbox with a minimal reproduction.

**Reply by grahampcharles** [reply]
Yeah, I was working on a repro, but it seems like an extreme edge case. The search box has a Kbd element in it that switches from ^K to ⌘K on Macs, which it detects by using the NavigatorWithUserAgentData interface. That seems to fail on iOS, but only in built apps, not off the dev server. I don't have an iOS test environment apart from building the app and opening it on my phone. So... it's a bit involved to test and _really_ easy to simply avoid. (Plus the app isn't really designed for iOS.)

Anyway, I think I'll just leave this comment here in case anyone else runs into the same issue -- I might troubleshoot it later but it's really not worth it to me.

**Comment by MichaelODeli** [comment]
@rtivital Just open mantine [headers sample page](https://ui.mantine.dev/category/headers/) and click on Burger button on headers.

**Reply by grahampcharles** [reply]
I was already using stock code, in fact. It was apparently the NavigatorWithUserAgentData call that bricked the entire javascript payload. Don't know why but don't particularly care -- I just removed it and all is fine. (And it's unrelated to Burger, per se.)

**Comment by Ashkumar7** [comment]
I can confirm this that there is issue with ios devices when clicking on the burger menu it doesnt open any model or drawer.
